File: src/main/java/plus/maa/backend/controller/request/comments/CommentsDeleteDTO.java
package plus.maa.backend.controller.request.comments;

import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author LoMu
 * Date  2023-02-19 10:50
 */

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CommentsDeleteDTO {
    @NotBlank(message = "评论id不可为空")
    private String commentId;
}

File: src/main/kotlin/plus/maa/backend/controller/request/comments/CommentsDeleteDTO.kt
package plus.maa.backend.controller.request.comments

import jakarta.validation.constraints.NotBlank

/**
 * @author LoMu
 * Date  2023-02-19 10:50
 */
data class CommentsDeleteDTO(
    @field:NotBlank(message = "评论id不可为空")
    val commentId: String
)